src/DDTrace/Integrations/SymfonyMessenger/DDTraceStamp.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace DDTrace\Integrations\SymfonyMessenger;
+
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+final class DDTraceStamp implements StampInterface
```

Comment:
```suggestion
/**
 * @deprecated since 1.5.0
 */
final class DDTraceStamp implements StampInterface
```

Shouldn't we mark it as deprecated?